validate.sh

```diff
@@ -0,0 +1,50 @@
+#! /bin/bash
+
+#=========================================================================
+# Copyright (C) GemStone Systems, Inc. 2010.
+#
+# Name - install.sh
+#
+# Purpose - Make sure system can run MagLev.
+#
+# $Id:$
+#
+# Description:
+#    Verify machine is capable of running MagLev 64-bit
+#
+# Actions:
+#    Verify machine is capable of running MagLev 64-bit
+#=========================================================================
+
+# Detect operating system
+PLATFORM="`uname -sm | tr ' ' '-'`"
```

Comment:
We have the following few lines at the top of `setup.sh` too. What's the best way to share the code? Should we have a `utils.sh` file that's sourced first?